microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -12,34 +12,37 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, I
 
 # Understand WebView2 SDK versions  
 
-WebView2 depends on Microsoft Edge to function.  Each WebView2 SDK requires that a minimum browser version is installed.  The minimum version is reflected in the package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install Microsoft Edge with a build number of 488 or later.  The browser version is also specified in the WebView2 [Release Notes][Releasenotes].  For more information about the latest release of the Microsoft Edge browser, navigate to [Browser Channels][DeployedgeChannels].  
+To develop a WebView2 application, a minimum version of [WebView2 Runtime]() or a [non-stable Microsoft Edge channel]() must be installed.  The minimum version is reflected in the package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install [WebView2 Runtime]() or a [non-stable Microsoft Edge channel]() with a build number of 488 or later.  The browser version is also specified in the WebView2 [Release Notes][Releasenotes].  For more information about the latest release of the Microsoft Edge browser, navigate to [Browser Channels][DeployedgeChannels].    
 
-> [!NOTE]
-> WebView2 is currently in preview.  While the WebView team strives to ensure backward compatibility between browser versions and SDKs, it is not guaranteed because newer versions of the browser may not support previous SDK versions.  If there are breaking changes between browser versions and SDKs, the changes are specified in the [Release Notes][Releasenotes].  
+New versions of the WebView2 SDK are shipped at the same cadence as the Microsoft Edge (Chromium) browser, which is approximately every 6 weeks. When a new SDK is shipped, it is compatible with the latest version of Microsoft Edge. Therefore, the WebView2 team recommends developing against the Microsoft Edge canary channel or the WebView2 Runtime. 
```

Comment:
`When a new SDK is shipped, it is compatible with the latest version of Microsoft Edge.` feel a little off since we have two SDKs with aligning with different things. Maybe talk about the SDKs in the ## Release and prerelease package section and specify which channel each SDK package align with? 

For reference, this is what we have in the GA blog post. You might be able copy or reword some of that into the docs.

> For those of you following along with WebView2, you may know that during our preview stage we aimed to release a new SDK every 6 weeks. Going forward, we’ll continue to be agile as both WebView2 SDK and Runtime updates will generally align with updates to Microsoft Edge Stable channel. You can expect that along with each browser update there will be an update to the evergreen WebView2 Runtime (and a new Fixed Version package available). A new WebView2 SDK with new APIs will also become available when a major browser/WebView2 Runtime version update (version 86, 87, etc.) is released. You don’t need to take advantage of every new SDK, but it’s a great way to always get the most out of what we’ve been working on. We also have two types of SDKs – “Release” SDKs that are forward compatible and meant to avoid any breaking changes, and “Prerelease” SDKs which includes additional experimental APIs for developers to try out and provide feedback on. We look forward to continuing to deliver updates to WebView2, and doing our best to add the features and fixes that you’ve been asking for.

Reply:
pre-release aligns with Canary and Release aligns with Dev?